src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/GetPileupSummaries.java

```diff
@@ -0,0 +1,99 @@
+package org.broadinstitute.hellbender.tools.walkers.contamination;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFConstants;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.VariantProgramGroup;
+import org.broadinstitute.hellbender.engine.FeatureContext;
+import org.broadinstitute.hellbender.engine.MultiVariantWalker;
+import org.broadinstitute.hellbender.engine.ReadsContext;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.utils.GATKProtectedVariantContextUtils;
+import org.broadinstitute.hellbender.utils.pileup.ReadPileup;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Given a sites vcf of common SNPs and a bam file, get summary of alt and ref counts along with population allele frequency.
+ *
+ * The output of this tool is the input for {@link CalculateContamination}.  Use them as follows:
+ *
+ * java -jar gatk.jar GetPileupSummaries -V ExAC.vcf -V thousand_genomes.vcf -I input.bam -O pileups.table
+ * java -jar gatk.jar CalculateContamination -I pileups.table -O contamination.table
+ *
+ * Created by David Benjamin on 2/15/17.
+ */
+@CommandLineProgramProperties(
+        summary = "Calculate pileup statistics for inferring contamination",
+        oneLineSummary = "Calculate pileup statistics for inferring contamination",
+        programGroup = VariantProgramGroup.class)
+public class GetPileupSummaries extends MultiVariantWalker {
+
+    public final String MAX_SITE_AF_LONG_NAME = "maximumPopulationAlleleFrequency";
+    public final String MIN_SITE_AF_LONG_NAME = "minimumPopulationAlleleFrequency";
+    public final String MAX_SITE_AF_SHORT_NAME = "maxAF";
+    public final String MIN_SITE_AF_SHORT_NAME = "minAF";
+
+    private static final double DEFAULT_MIN_POPULATION_AF = 0.01;
+    private static final double DEFAULT_MAX_POPULATION_AF = 0.2;
```

Comment:
Sure, maybe by talking it out we'll realize that the 20% ceiling in unnecessary.